coral-incremental/src/main/java/com/linkedin/coral/incremental/IncrementalTransformerResults.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.incremental;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.calcite.rel.RelNode;
+
+
+public class IncrementalTransformerResults {
+
+  private RelNode incrementalRelNode;
+  private RelNode refreshRelNode;
+  private Map<String, RelNode> intermediateQueryRelNodes;
```

Comment:
Should we do a map for deltas and a map for lasts?

Reply:
If we have 2 maps, then the keys may not require `_delta` and `_last` suffixes? Feel free to choose one that makes the implementation cleaner.

Reply:
+1 to splitting out deltas & last

Reply:
Sounds good, will split but maintain suffixes since I think this will come in handy on the code generation side.